service/version_bundles.go

```diff
@@ -292,5 +292,50 @@ func NewVersionBundles() []versionbundle.Bundle {
 			Version:      "2.0.0",
 			WIP:          false,
 		},
+		{
+			Changelogs: []versionbundle.Changelog{
+				{
+					Component:   "kvm-node-controller",
+					Description: "Updated KVM node controller with pod status bugfix.",
+					Kind:        versionbundle.KindChanged,
+				},
```

Comment:
So, should we also list it in Components?

Reply:
thing is, we don't have "versions" for our components, they are just tags
Also, I am not sure how much interesting information is that for the customer.

Reply:
Yes this is weird situation. Something to think about. Not interesting but we have that in the changelog which is customer facing.